activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -1040,7 +1040,7 @@ def connection_pool_list(role = ActiveRecord::Base.current_role)
       end
       alias :connection_pools :connection_pool_list
 
-      def establish_connection(config, owner_name: Base.name, role: ActiveRecord::Base.current_role, shard: Base.current_shard)
+      def establish_connection(config, owner_name: Base, role: ActiveRecord::Base.current_role, shard: Base.current_shard)
```

Comment:
There is a slight behavior change in this public API that now accepts the class instead of a string. We could defensively handle this but applications really should be using `establish_connection` on the handler for multiple databases and this API is new in 6.1 so I think it's ok to change it. Happy to handle a string if we disagree.